ui/event_handlers.go

```diff
@@ -25,7 +25,7 @@ const completedStatus = "✔"                // "●"
 const tileFormat = color.Bold
 
 var auxInfoFormat = color.HEX("#777777")
-var statusTitleTemplate = fmt.Sprintf(" %%s %%-%ds ", syftUI.StatusTitleColumn)
+var statusTitleTemplate = fmt.Sprintf(" %%s %%-%ds ", syftUI.StatusTitleColumn+4)
```

Comment:
@wagoodman is this ok? Background: increased it so new (longer) messages get aligned.

Reply:
please ignore